.github/workflows/test.yml

```diff
@@ -35,4 +35,4 @@ jobs:
       - name: Run tests
         working-directory: ./tests
         run: |
-          poetry run pytest qir_test.py
+          poetry run pytest parser_test.py generator_test.py
```

Comment:
Have all the tests for `qir_test.py` moved into `parser_test.py` and `generator_test.py`? If so, `qir_test.py` should be deleted; if not, it should be run!

Reply:
Ah, I see it has been renamed, nm.